packages/expo-image/src/utils/blurhash/decode.ts

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable */
+// @ts-nocheck
```

Comment:
Can we ignore only the problematic lines instead of skipping formatting and type checking for the whole file?

Reply:
Fair, I've just fixed the blurhash code instead